.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is.-->
+
+**To Reproduce**
+<!-- Steps to reproduce the behaviour:
+
+1. happens with input (include input files)
```

Comment:
'Input that triggers the problem' ?

Also, what do we do about bugs that occur through other routes (API, socket interface). How about 
'Example of how to trigger the bug (include self-contained input file(s) if possible)'